src/index.js

```diff
@@ -48,11 +40,10 @@ export default function(deckName) {
 
   models[top_model_id + ''] = model;
 
-  update('update col set models=:models where id=1', { ':models': JSON.stringify(models) });
+  exporter.update('update col set models=:models where id=1', { ':models': JSON.stringify(models) });
 
-  return {
-    addMedia,
-    addCard,
-    save
-  };
+  return ['addCard', 'addMedia', 'save'].reduce((prev, i) => {
```

Comment:
they are. We just  reexport not all `exporter` but only picked methods.

I'm not sure, you just don't export all methods before, and I try to not change external interface